tap_helpshift/client.py

```diff
@@ -165,7 +165,7 @@ async def _get_resp_text(resp):
 
                 try:
                     LOGGER.info('GET %s %r timeout=%s', url, params, timeout)
-                    async with self.session.get(url, params=params, timeout=timeout) as resp:
+                    async with self.session.get(url, params=params, timeout=(timeout and int(timeout))) as resp:
```

Comment:
I think we can handle this at the top level by ensuring that timeout is actually an int:
```
timeout = os.getenv('DEFAULT_HTTP_TIMEOUT')
try:
    DEFAULT_TIMEOUT = int(timeout) if timeout else None
except (TypeError, ValueError): # casting int can fail depending on the input
    DEFAULT_TIMEOUT = None
```